src/components/summary/SummaryComponent.tsx

```diff
@@ -192,9 +192,13 @@ export function SummaryComponent(_props: ISummaryComponent) {
         )}
       />
     );
-  } else if (layoutComponent?.getComponentType() === ComponentType.Presentation) {
+  } else if (
+    layoutComponent?.getComponentType() === ComponentType.Presentation &&
+    formComponentLegacy.type !== 'Summary' &&
+    formComponentLegacy.type !== 'Group'
+  ) {
```

Comment:
I don't think checking that the type is not `Summary` or `Group` is necessary here as they do not have a ComponentType, so they would fail because `null !== ComponentType.Presentation`.

Reply:
Checked now, and it's still needed. TypeScript doesn't know that `layoutComponent` and `formComponentLegacy` is supposed to be the same thing (in many ways), so checking the type of `layoutComponent` does not narrow the type of `formComponentLegacy` (which is used in the implementation below). So these checks will make sure to narrow the type of `formComponentLegacy` so that `GenericComponent` can render it.